tests/test_daikin.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+import logging
+
+from pydaikin.factory import DaikinFactory
+
+logging.basicConfig(level=logging.DEBUG, format='%(levelname)s: %(message)s')
+
+
+async def test_daikin():
```

Comment:
```suggestion
@pytest.mark.skip(reason="We don't require connection to real devices")
async def test_daikin():
```

https://docs.pytest.org/en/stable/how-to/skipping.html#skipping-test-functions

Even better would be to read IP from an ENVIRONMENT variable and skip if that env is not set, but this is good enough.